doc/api/cli.md

```diff
@@ -594,6 +595,17 @@ added: v0.11.15
 
 Specify ICU data load path. (Overrides `NODE_ICU_DATA`.)
 
+### `--import=module`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+Preload the specified module at startup.
+
+Follows [ECMAScript modules][] resolution rules.
+Use [`--require`][] to load a [CommonJS module][].
```

Comment:
I think we should document somewhere that `--import` always run after `--require` scripts no matter the order of the CLI flags.

Reply:
That seems unfortunate. I’d expect loading order to match argument order.

Reply:
That would be much harder to implement since `require` is synchronous and `import`/loading esm is asynchronous
In case order is important, I think using just `import` will work

Reply:
In that case, we probably shouldn't _allow_ both to be provided in a mixed order - iow, if there's a `--require` anywhere, all `--import` must come after the last one.

Reply:
I think there is currently no way to know in what order the `node` arguments were passed. We would also need to consider `NODE_OPTIONS`: `NODE_OPTIONS="--import a --require b" node --import c --require d script.js`

Reply:
If there was no way to know, how could we ensure require ordering now? (which we do)

Reply:
We only know the relative order for flags that allow multiple values (they are pushed in an array), but the C++ parser doesn't give us more information.